src/python/pants/backend/python/framework/stevedore/target_types.py

```diff
@@ -11,13 +11,13 @@
 
 
 class StevedoreNamespace(str):
-    f"""Tag a namespace in entry_points as a stevedore namespace.
+    """Tag a namespace in entry_points as a stevedore namespace.
 
     This is required for the entry_point to be visible to dep inference
     based on the `stevedore_namespaces` field.
 
     For example:
-        {PythonDistribution.alias}(
+        python_distribution(
             ...
             entry_points={{
```

Comment:
I think there's some doubled `{{` and `}}` that should become `{` / `}` now that there's no f-string

Reply:
🤦🏽